modules/users/tests/server/user.server.model.tests.js

```diff
@@ -436,17 +440,18 @@ describe('User Model Unit Tests:', function () {
     });
 
     it('should allow valid email address - "abc-def@abc.com"', function (done) {
-      var _user = new User(user);
+      var _user1 = new User(user1);
 
-      _user.email = 'abc-def@abc.com';
-      _user.save(function (err) {
+      _user1.email = 'abc-def@abc.com';
+      _user1.save(function (err) {
         if (!err) {
-          _user.remove(function (err_remove) {
+          _user1.remove(function (err_remove) {
+            should.not.exist(err);
```

Comment:
Correct, but I read is as line 447 `if (!err) {` (err does not exist) would never get to line 449 `should.not.exist(err);`